src/proxy/http/HttpConfig.cc

```diff
@@ -1080,6 +1080,7 @@ HttpConfig::reconfigure()
 
   params->outbound = m_master.outbound;
 
+  ats_free(params->proxy_hostname);
```

Comment:
The `params` variable is allocated with `new` on line 1077, so all of these pointers should not have anything assigned to them.  We discussed this at the review meeting and think this change is not necessary.

Reply:
Oh, I'm sorry for my mistake. I agree this change is not necessary.
I'm closing this now.